index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>mobile version</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header class="header">
```

Comment:
- [ ] [OPTIONAL] The hamburger button should be on the right and has a `nav` tag, please implement this it's a requirement

|Your design|Figma design|
|-|-|
|![Screenshot_20220528_164653](https://user-images.githubusercontent.com/25227022/170832718-94fd97c2-5e29-495f-bdf1-582070883581.png)|![Screenshot_20220528_165114](https://user-images.githubusercontent.com/25227022/170832870-476f0198-8a39-46e8-af1b-8ff0c36dea5c.png)|